src/react/hooks/use-room.ts

```diff
@@ -0,0 +1,101 @@
+import { ConnectionStatusChange, ErrorInfo, Room, RoomLifecycle, RoomStatusChange } from '@ably/chat';
```

Comment:
I'm either way. In other files (looking at use-chat-client and chat-client-provider) we've used the same style as in this file.

Reply:
That's a fair point, it was less a suggestion and more a thought for discussion - perhaps we should just stick as is for consistency and to avoid changing the other files for now :)